docs/pages/database-access/guides/mysql-self-hosted.mdx

```diff
@@ -176,8 +176,11 @@ a single self-hosted MySQL database:
 
 ```yaml
 teleport:
-  data_dir: /var/lib/teleport-db
+  data_dir: /var/lib/teleport
```

Comment:
This was actually intentional, for the use-case where you run db service on the same node as auth/proxy, so data directories wouldn't collide.

Reply:
For getting started guides should we just put it into one service? Auth/Proxy/DB? and for production / HA setups recommend a different data directory? It has a range of pros / cons for both approaches.

Reply:
I've left a comment so it's clear for new users just getting started with db access